building/trusted_applications.rst

```diff
@@ -373,35 +373,39 @@ third step the binary and its signature are stitched together into the full TA.
 Offline Signing of TAs
 ======================
 
-The TA dev kit does sign an application as last step of the linking process. For
-example, the file ``ta/arch/arm/link.mk`` in the :ref:`optee_os` source tree
-contains the statement
+There are two types of TAs that can be signed offline: the so-called `in-tree` TAs, which come
```

Comment:
I'd drop 'so-called' and also the "`" around in-tree, they are in-tree TAs.
If you're not happy with the name, what would you suggest?

Reply:
Dropped!